pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/PbjCompilerTask.java

```diff
@@ -120,6 +122,13 @@ public static void compileFilesIn(Iterable<File> sourceFiles,
                     }
                 } catch (Exception e) {
                     System.err.println("Exception while processing file: " + protoFile);
+                    // Print an abbreviated stack trace for help in debugging.
+                    System.err.println(e);
+                    var trace = e.getStackTrace();
+                    int count = 0;
+                    for (var element : trace) {
+                        if(count++ < MAX_TRACE_FRAMES) System.err.println(STACK_ELEMENT_INDENT + element);
```

Comment:
Minor: space after `if` please.

Reply:
We definitely need to get spotless on this repo...
done.